scala-package/core/src/main/scala/org/apache/mxnet/ExecutorManager.scala

```diff
@@ -479,15 +480,28 @@ private class DataParallelExecutorGroup private(sym: Symbol,
       trainData: DataIter,
       sharedGroup: DataParallelExecutorGroup) {
     this(sym, argNames, paramNames, ctx, slices,
-      trainData.provideData, trainData.provideLabel, sharedGroup)
+      trainData.provideDataDesc, trainData.provideLabelDesc, sharedGroup)
+  }
+
+  @deprecated("Use provideDataDesc and provideDataLabel instead", "1.3.0")
+  def this(sym: Symbol,
```

Comment:
If we find this class only be used internally, let's just kill the old param entirely.

Reply:
I didn't notice it was private. I'll make sure to remove all private deprecated code